### PR TITLE
Adding support for internationalization

### PR DESCRIPTION
**Pull request by Megala21**

### CODE REVIEW

File: components/application-mgt/org.wso2.carbon.device.application.mgt.publisher.ui/src/main/resources/publisher/public/locales/en.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
```
Comment (sinthuja) [inline components/application-mgt/org.wso2.carbon.device.application.mgt.publisher.ui/src/main/resources/publisher/public/locales/en.json:1]: Please add the license header.

File: components/application-mgt/org.wso2.carbon.device.application.mgt.publisher.ui/src/main/resources/publisher/src/components/Application/Create/CreateSteps/Step1.jsx [hunk 2]
```diff
@@ -131,47 +132,37 @@ class Step1 extends Component {
...
-                            <Input
-                                type="select"
-                                name="category"
-                                id="app-category"
-                            >
+                            <Label for="app-category">
+                                <FormattedMessage id='Category' defaultMessage='Category'/>
+                            </Label>
+                            <Input type="select" name="category" id="app-category">
                                 <option>Business</option>
```
Comment (sinthuja) [inline components/application-mgt/org.wso2.carbon.device.application.mgt.publisher.ui/src/main/resources/publisher/src/components/Application/Create/CreateSteps/Step1.jsx:151]: Isn't this shouldn't be used by localization?

**Review by sinthuja — CHANGES_REQUESTED** [review]
Please address the changes requested by the comments.

File: components/application-mgt/org.wso2.carbon.device.application.mgt.publisher.ui/src/main/resources/publisher/src/components/Application/Create/CreateSteps/Step1.jsx (hunk 2 above)
Comment (Megala21) [reply components/application-mgt/org.wso2.carbon.device.application.mgt.publisher.ui/src/main/resources/publisher/src/components/Application/Create/CreateSteps/Step1.jsx:151]: This is just a sample value. These should be actually taken back-end. So we do not need to add the localization here.

**Review by sinthuja — APPROVED** [review]

**Pull request merged**